bin/rubydoop-runner

```diff
@@ -0,0 +1,9 @@
+# encoding: utf-8
+
+require 'rubydoop.jar'
+require 'rubydoop/job_runner'
+
+require 'puck'
+$: << "classpath:#{Puck::Jar::JAR_APP_HOME}/lib"
```

Comment:
This is my main unsolved problem; how do I ensure the application's lib-directory is on the load path (so that I can load the job setup script) from the Rubydoop gem without using Puck's supposedly private constants?